otherlibs/win32unix/unix.ml

```diff
@@ -364,6 +364,7 @@ external isatty : file_descr -> bool = "unix_isatty"
 external unlink : string -> unit = "unix_unlink"
 external rename : string -> string -> unit = "unix_rename"
 external link : ?follow:bool -> string -> string -> unit = "unix_link"
+external realpath : string -> string = "unix_realpath"
```

Comment:
These paths get returned prefixed `\\?\` which needs removing (see the test)

There is also a difference in behaviour between Windows and Unix for files which you don't have access to: `realpath(3)` will still translate the path, but this won't. This might be an alternative:

```suggestion
external realpath : string -> string = "unix_realpath"

let realpath p =
  try realpath p
  with (Unix_error (EACCES, _, _)) as e ->
    try
      Filename.concat (realpath (Filename.dirname p)) (Filename.basename p)
    with _ -> raise e
```
(I've tested that - the _might_ refers to the use of `Filename.dirname` in this way)

Reply:
Done in 798f6f1